### PR TITLE
Added AlexNet MVCNN model and option for pretrained

### PR DESCRIPTION
**Pull request by zzuberi**
I have added a MVCNN model based on AlexNet. 

Also modified code to have pretrained option

### CODE REVIEW

File: models/resnet.py [hunk 1]
```diff
@@ -175,7 +175,11 @@ def resnet18(pretrained=False, **kwargs):
     """
     model = ResNet(BasicBlock, [2, 2, 2, 2], **kwargs)
     if pretrained:
-        model.load_state_dict(model_zoo.load_url(model_urls['resnet18']))
+        pretrained_dict = model_zoo.load_url(model_urls['resnet18'])
+        model_dict = model.state_dict()
+        pretrained_dict = {k: v for k, v in pretrained_dict.items() if k in model_dict and v.shape == model_dict[k].shape}
+        model_dict.update(pretrained_dict)
+        model.load_state_dict(model_dict)
```
Comment (RBirkeland) [inline models/resnet.py]: How is this different from before? As far as I understand you are just doing the same in a more complex way?

File: controller.py [hunk 2]
```diff
@@ -12,17 +12,24 @@
...
 
 from models.resnet import *
+from models.mvcnn import *
 import util
 from logger import Logger
 from custom_dataset import MultiViewDataSet
 
+MVCNN = 'mvcnn'
+RESNET = 'resnet'
+MODELS = [RESNET,MVCNN]
```
Comment (RBirkeland) [inline controller.py:22]: Both models are "mvcnn" but based on resnet and alexnet. If you call one resnet I would also call mvcnn alexnet.

Comment (zzuberi) [reply controller.py:22]: Yes you are correct

Comment (RBirkeland) [reply controller.py:22]: Anyway, thanks for your contributions, I merged it and did some minor fixes with the defaults :+1:

Comment (zzuberi) [reply controller.py:22]: Sure thanks for setting up the initial repo. It was very helpful.

**Pull request merged**